### DISCUSSION TITLE
Enabling HTTPS on drupal site, using orbstack as dover provider

### DISCUSSION THREAD

**Discussion by madamadami** [ddev/ddev#6503] [Q&A] [answered]
I installed Drupal 10.3.2 with the DDEV recipe, using [orbstack](https://docs.orbstack.dev/) as docker provider.

Unfortunately, if I want to browse https version of the Drupal site, I get the message in my browser that site isn't secure (net::ERR_CERT_COMMON_NAME_INVALID), but orbstack is [providing https for containers](https://docs.orbstack.dev/features/https) out-of-the-box and it works for non-Drupal projects flawlessly. I posted question on [SO](https://drupal.stackexchange.com/questions/320651/enabling-https-with-ddev-and-orbstack) but a user told me to turn to DDEV support instead.

What may cause the issue? What steps do I have to take in order to enable proper https version of my Drupal site?

**Comment by rfay** [comment, marked as answer]
Hi and Welcome @madamadami - DDEV has out-of-the-box support for TLS. Is there a reason you don't want to use it? You don't have to do anything but the one-time `mkcert -install` that is recommended in the docs.

It's fine that OrbStack provides other techniques, but DDEV's support works across all Docker providers and operating systems, so I haven't investigated trying its new technique. 

Please just do `mkcert -install` and then `ddev poweroff && ddev restart` and you should have TLS support right away.

**Reply by madamadami** [reply]
Thank you, `ddev poweroff && ddev restart` did the magic!
